### PR TITLE
Revamped comment layout and pagination

### PR DESCRIPTION
**Pull request by jonahlikescookies**
### Description

Addresses more changes in #16, a revamped layout for comments and pagination for comments.

### Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### Additional Information

Currently, pagination has not been implemented. Hopefully I should be able to work on it.

### CODE REVIEW

File: src/dashboard/public/styles/packs.css [hunk 1]
```diff
@@ -79,11 +79,42 @@ main {
     }
 
     #comments {
-        word-wrap: break-word;
-        
+        div.commentInfo {
+            display: flex;
```
Comment (MesterMan03) [inline src/dashboard/public/styles/packs.css:83]: text-align appears twice with different values, I assume it's not intentional

Comment (MesterMan03) [inline src/dashboard/public/styles/packs.css:83]: text-align appears twice with different values, I assume it's not intentional

Comment (jonahlikescookies) [reply src/dashboard/public/styles/packs.css:83]: Oops, sorry, I forgot to delete it when changing the text-align.

**Comment by MesterMan03** [comment]
@jonahlikescookies devdash mode will now generate 50 lorem ipsum comments for every pack, should help with testing.

**Comment by jonahlikescookies** [comment]
The `&EmptySquareSmall;` entity seems to look different across devices. If that's an issue, we can remove it, or we can find another way to put a square before the date.

**Comment by MesterMan03** [comment]
Yeah I've noticed that too, I think we should probably just remove it, since the date is in a new line anyway, that character was used to separate the username and date.

**Comment by jonahlikescookies** [comment]
I think just using a psuedo-element should work.

**Comment by jonahlikescookies** [comment]
Should the date on the comments have a localized format(e.g., mm/dd/yyyy and 12-hour time for en-US), or should it always be yyyy/mm/dd and 24-hour time?

**Comment by MesterMan03** [comment]
Since luxon is able to do localised dates, I'd suggest looking at https://moment.github.io/luxon/api-docs/index.html#datetimetolocalestring and just replace our custom format with it.

**Comment by MesterMan03** [comment]
Alright, I think this is complete. Once we merge into dashboard/packs, every task should be checked off and make it ready for public release.

**Comment by jonahlikescookies** [comment]
One thing that has been bugging me is that when you move to the next page, it jumps back to the top, so I have to scroll down if I want to go to the next page. I have a fix for this, if you want I can push the commit soon.

**Comment by MesterMan03** [comment]
Couldn't that be solved by moving the page selector to the top? Or would that introduce some UX problems?

**Comment by jonahlikescookies** [comment]
I think using the `HTMLElement.scrollIntoView()` method should work, except the scroll-margin isn't working for me, which results in the page select going all the way to the bottom. Even if we automatically scrolled down, if they want to read the comments, then they have to scroll back up and then down. It still creates the same problem. Maybe making it scroll up smoothly rather than instantly would help.

**Comment by MesterMan03** [comment]
My other suggestion would be to have two page selectors on the top and bottom, so there is no scrolling required when changing pages.

File: biome.json [hunk 2]
```diff
@@ -106,7 +106,7 @@
         "formatter": {
             "jsxQuoteStyle": "double",
             "quoteProperties": "asNeeded",
-            "trailingCommas": "none",
+            "trailingComma": "none",
```
Comment (MesterMan03) [inline biome.json]: trailingComma is deprecated in biome 1.8.3, which is what the project now uses
use `bun install @biomejs/biome@1.8.3` if you haven't upgraded it yet, even though it's in package.json

**Review by MesterMan03 — CHANGES_REQUESTED** [review]

File: biome.json (hunk 2 above)
Comment (jonahlikescookies) [reply biome.json]: Oh, sorry, I should've updated biome.

**Review by MesterMan03 — APPROVED** [review]
Okay well if you don't have anything else to add, it's ready.

**Pull request merged**